x-pack/plugins/fleet/public/components/agent_enrollment_flyout/managed_instructions.tsx

```diff
@@ -157,8 +175,9 @@ export const ManagedInstructions = React.memo<Props>(
 
     return (
       <>
-        {fleetStatus.missingRequirements?.length === 1 &&
-        fleetStatus.missingRequirements[0] === 'fleet_server' ? (
+        {(fleetStatus.missingRequirements?.length === 1 &&
+          fleetStatus.missingRequirements[0] === 'fleet_server') ||
+        fleetServers.length === 0 ? (
```

Comment:
nit: move to a named `const showFleetMissingRequirements`.

Should we make this more robust in case fleet_server is not the first one?

```ts
const showFleetMissingRequirements = fleetServers.length === 0 ||
  (fleetStatus.missingRequirements ?? []).some(r => r === 'fleet_server');
```

Reply:
done